subcommands/keys/rotate_root.go

```diff
@@ -91,6 +96,9 @@ func doRotateRoot(cmd *cobra.Command, args []string) {
 
 	newid, newPk, newCreds := swapRootKey(root, curid, creds)
 	fmt.Println("= New root:", newid)
+	root.Signed.Reason.PolisId = user.PolisId
+	root.Signed.Reason.Message = changeReason
```

Comment:
Do we want to limit a size of `changeReason`?

It looks a bit risky to put a user generated content into `root.json` since a user can put anything into that string.
Maybe, this is just my phobias :).

Reply:
The backend enforces a 120byte max-len